### ISSUE TITLE
Add functionality to open paths with spaces in iTerm2

### ISSUE THREAD

**Issue by jdeweese1** [Mortennn/FiScript#6]
I'm currently running into the issue where when Finder is open to a directory with whitespace in the path, such as my local`/Users/jaroddeweese/Library/Mobile Documents/com~apple~CloudDocs/`, I am unable to open the directory in iTerm2 with FiScript extension. This seems to be because `$PATH` is not escaped when passed to the script.

My terminal will open but prints the following message out:
```
cd /Users/jaroddeweese/Library/Mobile Documents/com~apple~CloudDocs/Documents; clear
cd: string not in pwd: /Users/jaroddeweese/Library/Mobile
```

The following is the relevant portion of my `Open iTerm Window` AppleScript script, my workaround and what I believe to be the solution:

```AppleScript
tell current session
write text "cd " & quoted form of $PATH & "; clear"
end tell
```
Note how I added __`quoted form of`__ to the script.

If this is the permanent fix to the problem, any other relevant places will need to be updated also, i.e `Open Terminal Window`, `Open Terminal Tab` etc.

**Comment by eduardoportilho** [comment]
Same issue here and your suggestion fixes it 👍

**Comment by Mortennn** [comment]
Fixed in #8
